events/store-vendor.js

```diff
@@ -0,0 +1,10 @@
+'use  strict';
+
+const events = require('events');
+
+const vendorName = process.env
+
+setInterval(() => {
+  let orderPlaced = {name: 'Fake Order', id: '01'}
+  events.EventEmitter('Order has been picked up!', order)
+}, 2000);
```

Comment:
after this line you can add a event.on('delivered', __a function__) where the function is console.log that say thank you for your order.